src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,121 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    private static final int TICKET_AMOUNT = 1000;
+    public static final int THREE_MATCH_RANK_REWARD = 5000;
+    public static final int FOUR_MATCH_RANK_REWARD = 50000;
+    public static final int FIVE_MATCH_RANK_REWARD = 1500000;
+    public static final int SIX_MATCH_RANK_REWARD = 2000000000;
+    private final List<Ticket> ticketList;
+    private int threeMatchRankCount = 0;
+    private int fourMatchRankCount = 0;
+    private int fiveMatchRankCount = 0;
+    private int sixMatchRankCount = 0;
```

Comment:
클래스변수, 인스턴스 변수들은 개행을 해주면 가독성이 높아지는 것 같은데 어떻게 생각하세요?
```suggestion
    public static final int THREE_MATCH_RANK_REWARD = 5000;
    public static final int FOUR_MATCH_RANK_REWARD = 50000;
    public static final int FIVE_MATCH_RANK_REWARD = 1500000;
    public static final int SIX_MATCH_RANK_REWARD = 2000000000;
    
    private static final int TICKET_AMOUNT = 1000;

    private final List<Ticket> ticketList;
    private int threeMatchRankCount = 0;
    private int fourMatchRankCount = 0;
    private int fiveMatchRankCount = 0;
    private int sixMatchRankCount = 0;
```

Reply:
훨씬 보기 좋은거 같습니다. 다음부터는 변수들 성격에 따라 개행을 넣도록 하겠습니다.